src/components/ReusableComponents/FollowUs/FollowUsIconsLight.jsx

```diff
@@ -0,0 +1,40 @@
+import FacebookLogoLight from "/icons/Facebook.png";
+import TwitterLogoLight from "/icons/TwitterX.png";
+import InstagramLogoLight from "/icons/Instagram.png";
+import { useTranslation } from "react-i18next";
+import styled from "styled-components";
```

Comment:
Is the Light mode being used? I guess its for maybe turning the icons lighter when you hover or something like that? I guess that is something you are planning on implementing in the future! Will be exciting to see!